internal/processing/istio/authorization_policy_processor.go

```diff
@@ -149,6 +149,13 @@ func generateAuthorizationPolicySpec(api *gatewayv1beta1.APIRule, rule gatewayv1
 }
 
 func withTo(b *builders.RuleBuilder, rule gatewayv1beta1.Rule) *builders.RuleBuilder {
+	if rule.Path == "/.*" {
```

Comment:
```suggestion
        // APIRule and VirtualService supported a regex match. Since AuthorizationPolicy supports only prefix, suffix and wildcard 
        // and we have clusters with "/.*" in APIRule, we need special handling of this case.
	if rule.Path == "/.*" {
```